website/docs/using-unleash/compliance/soc2.mdx

```diff
@@ -0,0 +1,32 @@
+
+This guide provides an overview of how Unleash features align with SOC2 controls, helping your organization meet its compliance requirements.
+ 
+
+## How Unleash features map to SOC2 controls
+
+| SOC2 Control                                | SOC2 Control Description                                                                                             | Unleash Feature                                                                                                                                           |
+|---------------------------------------------|---------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| CC 2.1, CC 7.2 Log management utilized      | The company utilizes a log management tool to identify events that may have a potential impact on the company's ability to achieve its security objectives. | Unleash provides a log of all [events](/reference/events), such as configuration changes and access.                                                                         |
+| CC 2.2, CC 5.3 Roles and responsibilities specified | Roles and responsibilities for the design, development, implementation, operation, maintenance, and monitoring of information security controls are formally assigned in job descriptions and/or the Roles and Responsibilities policy. | Unleash provides [role-based access control](/reference/rbac).                                                                                                                                          |
+| CC 2.2 System changes communicated          | The company communicates system changes to authorized internal users.                                               | Admins in Unleash can configure [banners](/reference/banners) that can display message for all users in the Unleash Admin UI.                                                                                                        |
+| CC 3.2, CC 7.5, CC 9.1 Continuity and disaster recovery plans tested | The company has a documented business continuity/disaster recovery (BC/DR) plan and tests it at least annually.       | Unleash provides a business continuity disaster recovery (BCDR) policy available to customers in the Trust Center, and annual test results upon request.                                                 |
+| CC 3.4, CC 7.1 Configuration management system established | The company has a configuration management procedure in place to ensure that system configurations are deployed consistently throughout the environment. | Unleash provides Release Plan Templates to implement consistent release sequences aligned with the customer policy. Additionally, the [Change Request](/reference/change-requests) supports 4-eyes approval workflows for changes. |
```

Comment:
I'm not sure about this one. On the one hand, I would not like to include features that are still in development. We should only put GA features here, I think. On the other hand, not sure how release templates (or change requests) would be a solution for managing 'system configuration'. If it's about the system configuration of the customer, then I think our general recommendation is that feature flags shouldn't manage configuration. If it's about the system configuration of Unleash, then not sure how change requests or release templates help with that.

If we're talking about the Unleash configuration/setup, then I think import/export and terraform are a better fit here.

Reply:
Configuration management here is a general term for the state of the system.  It is asking if you have a process in place to management system changes. In the case of feature flags that control production state, that is configuration.  So this is ok.  WE can remove the Release plans part for now. Can you remember to add back once we have rolled out?

Reply:
I agree to remove Release Plans for now, and add it when it reaches GA